deploy/base/deployment.yaml

```diff
@@ -114,26 +109,21 @@ spec:
             exec:
               command: ["/bin/grpc_health_probe", "-addr=:12345"]
             periodSeconds: 5
-          livenessProbe:
-            exec:
-              command: ["/bin/grpc_health_probe", "-addr=:12345"]
-            periodSeconds: 5
-            initialDelaySeconds: 10
+            failureThreshold: 60
+          # livenessProbe:
```

Comment:
Add a note here on why it's commented out?

Reply:
Done